crates/iota-framework/packages/iota-system/sources/validator_set.move

```diff
@@ -1129,4 +1490,153 @@ module iota_system::validator_set {
         };
         res
     }
+
+    public(package) fun committee_validator_addresses(self: &ValidatorSetV2): vector<address> {
+        let vs = &self.active_validators;
+        let num_validators = vs.length();
+        let committee_members = &self.committee_members;
+
+        let mut res = vector[];
+        let mut i = 0;
+        let committee_members_num = committee_members.length();
+        while (i < committee_members_num) {
+            let validator_index = committee_members[i];
+            assert!(
+                validator_index < num_validators,
+                ECommitteeMembersSetCorrupt,
+            );
+            let validator_address = vs[validator_index].iota_address();
+            res.push_back(validator_address);
+            i = i + 1;
+        };
+        res
+    }
+
+    // Selects new set committee members from the set of all active validators.
+    // Currently selects all active validators to be part of the committee.
+    public(package) fun select_committee_members(self: &mut ValidatorSetV2) {
+        let mut new_committee_members = vector[];
+        let active_validators = &self.active_validators;
+        let active_validators_num = active_validators.length();
+        
+        let mut i = 0;
+        while (i < active_validators_num) {
+            new_committee_members.push_back(i);
+            i = i + 1;
+        };
+
+        self.committee_members = new_committee_members;
+    }
+
+
+    // Selects top N stakers among all active validators to be part of the committee.
+    public(package) fun top_n_stakers(self: &mut ValidatorSetV2, n: u64) {
```

Comment:
I would maybe call it `select_committee_members_ top_n_stakers` or something?
Or at least `select_top_n_stakers`.

Reply:
Good point.

Reply:
Or just make this function `top_n_stakers(self: &ValidatorSetV2, n: u64): vector<u64>` and make the final assignment at the call site, there's just two places where this function gets called.